### PR TITLE
Add empty hand-written session management client library as stage for work

### PR DESCRIPTION
**Pull request by jonathanmendez**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Adds a library into which we will be adding upcoming classes and VIs. This library will be a hand-written, user-facing library that houses the improved APIs for session management.
The library is added to the Measurement Server.lvproj that houses the existing hand-written code.

There already exists a tool-generated "MeasurementLink Session Management Client" library, which needs to keep its existing name to maintain backward compatibility. This library adds the "NI " prefix to its desired name, which we may decide to use for all future-state libraries anyway, but creates a naming distinction.

### Why should this Pull Request be merged?

By staging the empty library, we can enable parallel work with easier merge conflicts.

### What testing has been done?

None.

### CODE REVIEW

File: Source/Runtime/NI MeasurementLink Session Management Client/NI MeasurementLink Session Management Client.lvlib [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
... 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 0 0 0 0 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 0 0 0 0 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 255 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 0 0 0 7 86 73 32 73 99 111 110 100 1 0 0 0 0 0 5 115 101 115 115 110 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 11 83 109 97 108 108 32 70 111 110 116 115 0 1 8 1 1
```
Comment (jonathanmendez) [inline Source/Runtime/NI MeasurementLink Session Management Client/NI MeasurementLink Session Management Client.lvlib:7]: image

Comment (pbirkhol-ni) [reply Source/Runtime/NI MeasurementLink Session Management Client/NI MeasurementLink Session Management Client.lvlib:7]: I prefer Sess of Sessn.

File: Source/Runtime/Measurement Server.lvproj [hunk 2]
```diff
@@ -14,6 +14,7 @@
 		<Property Name="specify.custom.address" Type="Bool">false</Property>
 		<Item Name="MeasurementLink Measurement Server" Type="Folder">
 			<Item Name="MeasurementLink Measurement Server.lvlib" Type="Library" URL="../MeasurementLink Measurement Server/MeasurementLink Measurement Server.lvlib"/>
+			<Item Name="NI MeasurementLink Session Management Client.lvlib" Type="Library" URL="../NI MeasurementLink Session Management Client/NI MeasurementLink Session Management Client.lvlib"/>
```
Comment (pbirkhol-ni) [inline Source/Runtime/Measurement Server.lvproj:17]: I won't argue the name since I can't come up with a better alternative.  :)

**Review by pbirkhol-ni — APPROVED** [review]

File: Source/Runtime/NI MeasurementLink Session Management Client/NI MeasurementLink Session Management Client.lvlib (hunk 1 above)
Comment (jonathanmendez) [reply Source/Runtime/NI MeasurementLink Session Management Client/NI MeasurementLink Session Management Client.lvlib]: Updated

Comment (jasonmreding) [reply Source/Runtime/NI MeasurementLink Session Management Client/NI MeasurementLink Session Management Client.lvlib]: I don't think we expect to have any VIs that are directly owned by the library so I don't think this banner will be visible anywhere other than the properties page for the library. However, it doesn't hurt to have it.

File: Source/Runtime/Measurement Server.lvproj (hunk 2 above)
Comment (jasonmreding) [reply Source/Runtime/Measurement Server.lvproj:17]: I don't know why, but tacking "NI" to the front of the library name when nothing else has it really bothers me. I would suggest just dropping "Client" from the library name for now until we have a better plan in place to fix all of the naming inconsistencies. i.e. `MeasurementLink Session Management.lvlib`.

**Review by jasonmreding — APPROVED** [review]

File: Source/Runtime/Measurement Server.lvproj (hunk 2 above)
Comment (jonathanmendez) [reply Source/Runtime/Measurement Server.lvproj]: That sounds better for now. Implemented.

**Pull request merged**